Assets/MRTK/Providers/WindowsMixedReality/XR2018/Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.asmdef

```diff
@@ -3,7 +3,9 @@
     "references": [
         "Microsoft.MixedReality.Toolkit",
         "Microsoft.MixedReality.Toolkit.Gltf",
-        "Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.Shared"
+        "Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.Shared",
+        "Microsoft.MixedReality.Toolkit.Services.InputSimulation",
```

Comment:
It feels off to me that this assembly would need to know about input simulation. Doing it this way would also mean we'd need a corresponding fix in the XR SDK version of this assembly, and any future platforms that support hands in the editor (Leap Motion might exhibit this same issue).